File: MyClassLibrary/Forms/WaitForm1.cs
using System;

namespace MyClassLibrary.Forms
{
	public partial class WaitForm1
	{
		public WaitForm1()
		{
			InitializeComponent();
			progressPanel.AutoHeight = true;
		}

		public override void SetCaption(string caption)
		{
			base.SetCaption(caption);
			progressPanel.Caption = caption;
		}

		public override void SetDescription(string description)
		{
			base.SetDescription(description);
			progressPanel.Description = description;
		}

		public override void ProcessCommand(Enum cmd, object arg)
		{
			base.ProcessCommand(cmd, arg);
		}
	}
}

File: MyClassLibrary/Forms/WaitForm1.vb
Namespace Forms

    Public Class WaitForm1

        Sub New()
            InitializeComponent()
            Me.progressPanel.AutoHeight = True
        End Sub

        Public Overrides Sub SetCaption(ByVal caption As String)
            MyBase.SetCaption(caption)
            Me.progressPanel.Caption = caption
        End Sub

        Public Overrides Sub SetDescription(ByVal description As String)
            MyBase.SetDescription(description)
            Me.progressPanel.Description = description
        End Sub

        Public Overrides Sub ProcessCommand(ByVal cmd As System.Enum, ByVal arg As Object)
            MyBase.ProcessCommand(cmd, arg)
        End Sub

    End Class

End Namespace